### DISCUSSION TITLE
all latest release zip files "Not Found" — is this to be expected?

### DISCUSSION THREAD

**Discussion by Luz-Angel** [stakira/OpenUtau#964] [Q&A]
I was looking for vocaloid alternatives and found OpenUtau listed, but I can't download the latest release (for windows) because it's "Not Found". Same for all latest release versions. I was wondering if this is to be expected as of late (2023 Dec 1).

**Comment by nagotown** [comment]
Is this still occurring? It seems to be working at the moment.
